libs/designsystem/form-field/src/input/input.component.spec.ts

```diff
@@ -142,18 +142,22 @@ describe('InputComponent', () => {
       });
     });
   });
-});
```

Comment:
moved the existing tests inside the top level describe block to avoid duplicate setup